packages/node/src/events/propose-install/controller.ts

```diff
@@ -0,0 +1,17 @@
+import { RequestHandler } from "../../request-handler";
+import { ProposeMessage } from "../../types";
+
+import { setAppInstanceIDForProposeInstall } from "./operation";
+
+export async function proposeInstallEventController(
+  this: RequestHandler,
```

Comment:
The pattern of using the variable name `this` when its a bit of a lie / hack seems like it could be a problem. Perhaps just name the variable `requestHandler`?

Reply:
Sure